sirepo/package_data/static/js/cloudmc.js

```diff
@@ -2978,6 +2978,21 @@ SIREPO.app.directive('tallyList', function() {
 });
 
 SIREPO.viewLogic('tallySettingsView', function(appState, cloudmcService, panelState, utilities, $scope) {
+    const useCachedScore = () => {
```

Comment:
This can be simplified quite a bit. First, `cloudmcService` has various `find()` functions, to which add `findScore`, like
```javascript
self.findScore = (score) => findScore(
    appState.models.openmcAnimation.tallies,
    appState.models.openmcAnimation.tally,
   score
);
```
Then you can just do
```javascript
const useCachedScore = () => {
    const s = cloudmcService.findScore(appState.models.openmcAnimation.prevScore);
    if (cloudmcService.findScore(s)) {
        appState.models.openmcAnimation.score = s;
    }
};
```

Reply:
Thats great. Thank you